### DISCUSSION TITLE
How do you get the TextBlock character clicked on?

### DISCUSSION THREAD

**Discussion by MarkBrainfire** [AvaloniaUI/Avalonia#12672] [General]
Is there a way to get the character or character index clicked on within a TextBlock?

**Comment by workgroupengineering** [comment]
https://github.com/AvaloniaUI/Avalonia/blob/f6809ad4e6b1f68d7cb1d797e5bd0d147910ce36/src/Avalonia.Controls/SelectableTextBlock.cs#L246-L248

**Reply by MarkBrainfire** [reply]
Thanks.  That looks simple and obvious.  Uno Platform has NO WAY of detecting the text a user clicks on, which is unbelievable to me!  I'm switching my entire project to Avalonia UI!
